airbyte-integrations/connectors/source-bigcommerce/source_bigcommerce/schemas/orders.json

```diff
@@ -9,8 +9,7 @@
       "type": ["null", "string"]
     },
     "date_shipped": {
-      "type": ["null", "string"],
-      "format": "date-time"
+      "type": ["null", "string"]
```

Comment:
@marcosmarxm what would be the impact of this change for normalization? This column won't be considered as a date and this might impact the users of this connector after an upgrade. A more global question: how does the normalization handle changes on the schema?